### PR TITLE
Fix update on flying request

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

**Review by dangleptr — COMMENTED** [review]
Weird.  IIRC, the issue has been fixed in your another pr merged.

**Comment by darionyaphet** [comment]
> Weird. IIRC, the issue has been fixed in your another pr merged.

Sorry to say `ADD` and `DELETE` have fixed but `UPDATE` missed.

**Comment by dutor** [comment]
Please address the build failure.

File: src/storage/exec/UpdateNode.h [hunk 1]
```diff
@@ -496,13 +489,13 @@ class UpdateEdgeNode : public UpdateNode<cpp2::EdgeKey> {
...
                         return folly::none;
                     }
-                    return this->updateAndWriteBack(partId, edgeKey, requestPtr);
+                    return this->updateAndWriteBack(partId, edgeKey);
                 } else {
                     // If filter out, StorageExpressionContext is set in filterNode
                     return folly::none;
                 }
             },
-            [&ret, requestPtr, &baton, this] (kvstore::ResultCode code) {
```
Comment (panda-sheep) [inline src/storage/exec/UpdateNode.h:505]: Missing `  planContext_->env_->onFlyingRequest_.fetch_sub(1);`   ？

**Review by dutor — APPROVED** [review]
LGTM.

**Review by panda-sheep — APPROVED** [review]
LGTM

**Pull request merged**